pkgs/top-level/all-packages.nix

```diff
@@ -16305,6 +16305,16 @@ with pkgs;
     ];
   });
 
+  libgit2_1_1 = libgit2.overrideAttrs (oldAttrs: rec {
```

Comment:
We should only create a top-level entry if many packages need that override. If only gitaly needs it, it can be done locally.

Reply:
Just ran into the same libgit issue with turbogit, should I add the same override locally or build on top of this PR?

Reply:
Can confirm that turbogit works with the override: ```  turbogit = callPackage ../development/tools/turbogit {
    # git2go only supports v1.1.x
    libgit2 = libgit2_1_1;
  };```

Reply:
Then I guess we can add a top-level entry if other packages need it. You can create a PR with the change when this is merged.

Reply:
Turbogit PR: https://github.com/NixOS/nixpkgs/pull/145068